### ISSUE TITLE
Using trained DAE

### ISSUE THREAD

**Issue by sandeepnmenon** [blackecho/Deep-Learning-TensorFlow#58]
After setting up the package and running the following command
<code>python run_autoencoder.py --n_components 1024 --batch_size 64 --num_epochs 20 --verbose 1 --learning_rate 0.05 --corr_type masking --corr_frac 0.5 --enc_act_func sigmoid --dec_act_func sigmoid --loss_func cross_entropy --opt momentum --momentum 0.9 --main_dir dae-models --model_name dae1024
</code>
The folder .yadlt was created with folders 'data', 'logs' and 'models'
The 'models' folder has 'checkpoint ' , 'dae.data-00000-of-00001' ,  'dae.index'  and 'dae.meta' files in it.
Did the code run properly?
If yes where is the trained model?
How to use the trained model to denoise images?

Thank You
